README.md

```diff
@@ -1,2 +1,242 @@
-# learn-user-testing
-Learn how to test your app with real humans 
+# Learn User Experience Testing
+Learn how to test your app with real humans
+
+## Why?
+This is _probably_ the most important/useful skill any "_creative technologist_"
+has. 💪
+
+Well conducted user experience testing helps you ensure that what you are
+planning to build is going to solve the problem for your users that you set out
+to solve. ✅
+
+It prevents you from wasting time ⏳/ effort 😓/ money 💰on guessing 🤔❓what your
+users will want, need or understand by getting them to critique your product as
+you go rather than retrospectively.
+
+Testing usability and following usability guidelines is not something we have to
+do to restrict ourselves or because we're told to. Understanding usability is
+about ensuring users can use our product effectively and so without it our
```

Comment:
Consider rephrasing to remove _double-negative_. "without ... less successful" ...
if you just cut the sentence at "_effectively_" it does not lose any meaning.